docs/frequently-asked-questions.rst

```diff
@@ -666,6 +666,22 @@ gas and return your 20 Wei).
 In the above example, the low-level function `call` is used to invoke another
 contract with `p.data` as payload and `p.amount` Wei is sent with that call.
 
+What happens to a mapping copied from a mapping of structs?
+===========================================================
+
+This is a very interesting question. Suppose that we have a contract field set up like such::
+
+    struct user{
+        mapping(string => address) usedContracts;
+    }
+    mapping(uint => user) userList;
+    function somefunction{
+       user userA = userList[1];
```

Comment:
Can you change that to `userList[2] = userList[1]`? Perhaps it would even be betten not to use two mappings here but rather state variables `user u1; user u2` and then `u1 = u2`?